Digipost/Logger.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Logger.swift
```

Comment:
Can you turn off these headers in the editor? No need to tell ourselves which file we're in